lib/readyset.rb

```diff
@@ -13,8 +13,17 @@ def self.configuration
     @configuration ||= Configuration.new
   end
 
+  def self.current_configuration
```

Comment:
IMO, this method is prob not super necessary -- users can always run `#inspect` on our config if they want to

Reply:
Agreed. Bit redundant now that I look at it.